spec/rails_helper.rb

```diff
@@ -66,6 +66,17 @@
   # config.filter_gems_from_backtrace("gem name")
 
   config.include FactoryBot::Syntax::Methods
+
```

Comment:
is there a reason these database cleaner lines are commented out?

Reply:
Its not working the way I want it to